bench/perf_run_py/perf_config.py

```diff
@@ -58,7 +58,8 @@ def __init__(self,
                  run_max: int = 1,
                  verbose: bool = False,
                  git_root: str = None,
-                 json_info=None):
+                 json_info=None,
+                 opts_flag=None):
```

Comment:
typo: We've got `opt_flags` in a few files that should be updated to `improved_accuracy`.